client/src/components/App/Admin/Bookings.tsx

```diff
@@ -62,25 +62,23 @@ const Bookings: React.FC<RouteComponentProps> = () => {
   const fullScreen = useMediaQuery(theme.breakpoints.down('sm'));
 
   // Helpers
-  const getAllOffices = useCallback(() => {
-    getOffices()
-      .then((data) =>
-        // Store in global state
-        dispatch({
-          type: 'SET_OFFICES',
-          payload: data,
+  const getAllBookings = useCallback(() => {
+    if (selectedOffice) {
+      getBookings({ office: selectedOffice.name, date: format(selectedDate, 'yyyy-MM-dd') })
+        .then((data) => {
+          setAllBookings(data.filter((booking) => !isPast(endOfDay(new Date(booking.date)))));
```

Comment:
I don' think we want this filter - why would we want to hide bookings that are served from the API? 

I know this PR is just maintaining the current behaviour, but I think this would be a good point to remove this filter unless there's a reason to keep it.

Reply:
This wasn't my coding, but I agree, I set up the filters to look historically.